docs/devices/371222402.md

```diff
@@ -0,0 +1,34 @@
+<!-- Notes BEGIN -->
+>> ## Notes
+---
+title: "Paulmann Licht 371222402 control via MQTT"
+description: "Integrate your Paulmann Licht 371222402 via Zigbee2MQTT with whatever smart home infrastructure you are using without the vendor's bridge or gateway."
+addedAt: 2025-03-14T15:30:00Z
+pageClass: device-page
+---
+
+# Paulmann Licht 371222402
+
+|     |     |
+|-----|-----|
+| Model | 371222402  |
+| Vendor  | Paulmann Licht  |
+| Description | Puric Pane pendant light 6x6W dimmable |
+| Exposes | light (state, brightness), effect, power_on_behavior |
+| Picture | ![Paulmann Licht 371222402](https://www.zigbee2mqtt.io/images/devices/371222402.png) |
+
+const m = require('zigbee-herdsman-converters/lib/modernExtend');
+
+const definition = {
```

Comment:
Could you make a PR by clicking [here](https://github.com/Koenkk/zigbee-herdsman-converters/edit/master/src/devices/paulmann.ts)?